osidb/models.py

```diff
@@ -559,6 +559,28 @@ class FlawMajorIncident(models.TextChoices):
 
         INVALID = "INVALID"
 
+    class FlawNistCvssValidation(models.TextChoices):
+        """
+        This flag determines the status of feedback loop between NIST and RH
+        requesting NVD CVSSv3 rescore for the CVE of this flaw.
+        The flag states have the following meaning:
+
+        * ( ) <unset>: No CVSSv3 rescore request was sent to NIST.
```

Comment:
Please update `<unset>` to `"novalue"`

Reply:
Fixed in a different way than you asked

- updated `<unset>` to `"" (no value)`
- updated lowercase `"requested"` etc. to uppercase `"REQUESTED"` etc

The reason is that the quoted words in the docstring started to be ambiguous in the terms of whether it's just a commentary or whether it's the exact string value. So I updated them to exactly match the enum choices to prevent ambiguity.